### PR TITLE
Add java doc for LightManager

### PR DESCRIPTION
**Pull request by pixelflinger**

### CODE REVIEW

File: android/filament-android/src/main/java/com/google/android/filament/LightManager.java [hunk 1]
```diff
@@ -20,72 +20,299 @@
 import androidx.annotation.Nullable;
 import androidx.annotation.Size;
 
+/**
+ * LightManager allows to create a light source in the scene, such as a sun or street lights.
```
Comment (prideout) [inline android/filament-android/src/main/java/com/google/android/filament/LightManager.java:24]: allows you

File: android/filament-android/src/main/java/com/google/android/filament/LightManager.java [hunk 2]
```diff
@@ -20,72 +20,299 @@
...
+ */
 public class LightManager {
     private long mNativeObject;
 
     LightManager(long nativeLightManager) {
         mNativeObject = nativeLightManager;
     }
 
+    /**
+     * Returns the number of component in the LightManager, note that component are not
```
Comment (prideout) [inline android/filament-android/src/main/java/com/google/android/filament/LightManager.java:126]: component => components (twice)

File: android/filament-android/src/main/java/com/google/android/filament/LightManager.java [hunk 3]
```diff
@@ -95,72 +322,244 @@ public Builder shadowOptions(@NonNull ShadowOptions options) {
...
+         * @return This Builder, for chaining calls.
+         */
         @NonNull
         public Builder falloff(float radius) {
             nBuilderFalloff(mNativeBuilder, radius);
             return this;
         }
 
+        /**
+         * Defines a spot light'st angular falloff attenuation.
```
Comment (prideout) [inline android/filament-android/src/main/java/com/google/android/filament/LightManager.java:483]: spelling

File: android/filament-android/src/main/java/com/google/android/filament/LightManager.java [hunk 4]
```diff
@@ -189,91 +588,308 @@ public Type getType(@EntityInstance int i) {
...
+     * @param efficiency    Efficiency in percent. This depends on the type of lightbulb used.
+     *
+     * @return This Builder, for chaining calls.
+     */
     public void setIntensity(@EntityInstance int i, float watts, float efficiency) {
         nSetIntensity(mNativeObject, i , watts, efficiency);
     }
 
+    /**
+     * returns the light's luminous intensity in lumen.
```
Comment (prideout) [inline android/filament-android/src/main/java/com/google/android/filament/LightManager.java:766]: lumen => lumens.  returns => Returns

File: android/filament-android/src/main/java/com/google/android/filament/LightManager.java [hunk 5]
```diff
@@ -189,91 +588,308 @@ public Type getType(@EntityInstance int i) {
...
+     * @param z Light's position z coordinate in world space. The default is 0.
+     *
+     * @see Builder#position
+     */
     public void setPosition(@EntityInstance int i, float x, float y, float z) {
         nSetPosition(mNativeObject, i, x, y, z);
     }
 
+    /**
+     * returns the light's position in world space
```
Comment (prideout) [inline android/filament-android/src/main/java/com/google/android/filament/LightManager.java:643]: In several places, "returns" should be "Returns"

File: android/filament-android/src/main/java/com/google/android/filament/LightManager.java (hunk 1 above)
Comment (pixelflinger) [inline android/filament-android/src/main/java/com/google/android/filament/LightManager.java:24]:
```suggestion
 * LightManager allows you to create a light source in the scene, such as a sun or street lights.
```

File: android/filament-android/src/main/java/com/google/android/filament/LightManager.java (hunk 2 above)
Comment (pixelflinger) [inline android/filament-android/src/main/java/com/google/android/filament/LightManager.java:126]:
```suggestion
     * Returns the number of components in the LightManager, note that components are not
```

File: android/filament-android/src/main/java/com/google/android/filament/LightManager.java (hunk 3 above)
Comment (pixelflinger) [inline android/filament-android/src/main/java/com/google/android/filament/LightManager.java:483]:
```suggestion
         * Defines a spot light's angular falloff attenuation.
```

File: android/filament-android/src/main/java/com/google/android/filament/LightManager.java (hunk 4 above)
Comment (pixelflinger) [inline android/filament-android/src/main/java/com/google/android/filament/LightManager.java:766]:
```suggestion
     * returns the light's luminous intensity in <i>lumens</i>.
```

File: android/filament-android/src/main/java/com/google/android/filament/LightManager.java [hunk 6]
```diff
@@ -20,72 +20,299 @@
...
+     *         The instance is 0 if the component doesn't exist.
+     * @see #hasComponent
+     */
     @EntityInstance
     public int getInstance(@Entity int entity) {
         return nGetInstance(mNativeObject, entity);
     }
 
+    /**
+     * destroys this component from the given entity
```
Comment (bejado) [inline android/filament-android/src/main/java/com/google/android/filament/LightManager.java:157]:
```suggestion
     * Destroys this component from the given entity
```

**Pull request merged**